src/ui/components/ListaProductos/icons/HomeIcon.jsx

```diff
@@ -0,0 +1,13 @@
+import { Icon } from '@mui/material';
+
+function HomeIcon(props) {
+  return (
+    <Icon {...props}>
+        <svg width="20px" height="20px">
```

Comment:
don't copy all svg code in html plain text. you can export the file into the assets and use it directly, like this.you can use 

```
import React from 'react';
import SvgIcon from '@material-ui/core/SvgIcon';
import { ReactComponent as ImportedSvg } from '../assets/logo.svg';

const MyComponent = () => {
  return (
    <SvgIcon component={ImportedSvg} viewBox="0 0 24 24" />
  );
};

export default MyComponent;
```